### PR TITLE
Windows: Enable multiple TestInspect tests

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard jhoward@microsoft.com

Enables a bunch more `TestInspect*` tests on Windows CI.

### CODE REVIEW

File: integration-cli/docker_cli_inspect_test.go [hunk 1]
```diff
@@ -79,19 +81,17 @@ func (s *DockerSuite) TestInspectStatus(c *check.C) {
 }
 
 func (s *DockerSuite) TestInspectTypeFlagContainer(c *check.C) {
-	testRequires(c, DaemonIsLinux)
 	//Both the container and image are named busybox. docker inspect will fetch container
 	//JSON State.Running field. If the field is true, it's a container.
 
-	dockerCmd(c, "run", "--name=busybox", "-d", "busybox", "top")
+	dockerCmd(c, "run", "--name=busybox", "-d", "busybox", "sleep", "60")
```
Comment (icecrime) [inline integration-cli/docker_cli_inspect_test.go]: Isn't this what the helper function `runSleepingContainer()` does?

File: integration-cli/docker_cli_inspect_test.go [hunk 2]
```diff
@@ -373,8 +368,15 @@ func (s *DockerSuite) TestInspectStopWhenNotFound(c *check.C) {
 }
 
 func (s *DockerSuite) TestInspectHistory(c *check.C) {
+	// This is still problematic on Windows - race condition in test
 	testRequires(c, DaemonIsLinux)
-	dockerCmd(c, "run", "--name=testcont", "-d", "busybox", "top")
+	id, _ := dockerCmd(c, "run", "--name=testcont", "-d", "busybox", "echo", "hello")
```
Comment (icecrime) [inline integration-cli/docker_cli_inspect_test.go]: Could we remove `-d` and avoid the need for waiting?

File: integration-cli/docker_cli_inspect_test.go (hunk 1 above)
Comment (jhowardmsft) [inline integration-cli/docker_cli_inspect_test.go]: Good point. Don't know why I didn't notice that as I updated so many others yesterday to do exactly that... Will be in next push.

File: integration-cli/docker_cli_inspect_test.go (hunk 2 above)
Comment (jhowardmsft) [inline integration-cli/docker_cli_inspect_test.go]: Good point. I'll update.

**Comment by jhowardmsft** [comment]
@icecrime - comments addressed and tests updated :smile:

**Comment by tonistiigi** [comment]
LGTM

**Comment by vdemeester** [comment]
and it's geen, yay @jhowardmsft 
LGTM 🐸

**Pull request merged**